queue_services/entity-filer/Makefile

```diff
@@ -61,7 +61,7 @@ install-dev: ## Install local application
 #################################################################################
 ci: lint flake8 test ## CI flow
 
-lint: ## Linting with pylint
```

Comment:
we're moving to a target of just lint ... but people keep mucking with this.
please just make it lint, which should call both pylint & flake8 targets